android-pay/src/main/java/com/stripe/wrap/pay/AndroidPayConfiguration.java

```diff
@@ -103,6 +110,37 @@ public static AndroidPayConfiguration init(
         mUsesSources = shouldUseSources;
     }
 
+    /**
+     * Specifies a set of card networks to allow. If not explicitly set,
+     * the default supported networks will be AMEX, DISCOVER, MASTERCARD, and VISA.
+     *
+     * @param networks one or more {@link WalletConstants.CardNetwork} values
+     */
+    public void addAllowedCardNetworks(@WalletConstants.CardNetwork Integer... networks) {
```

Comment:
Pinged you async, but is it possible to remove the added complexity of getting their card brands and using those for the user?  Could we have a nicer version that accepts either our card brands or just an array of strings?